cmd/crank/push.go

```diff
@@ -75,6 +79,22 @@ func (c *pushCmd) Run(child *pushChild, logger logging.Logger) error {
 		logger.Debug("Failed to create image from package tarball", "error", err)
 		return err
 	}
+
+	sourceDateEpoch := os.Getenv("SOURCE_DATE_EPOCH")
```

Comment:
Can we make this a flag instead? I believe Kong (our CLI library) allows you to specify `env:"SOURCE_DATE_EPOCH"` as a struct tag. This way you'd still be able to set this as an environment variable, but it would be a bit more discoverable and better documented.